app/models/post.rb

```diff
@@ -15,5 +15,4 @@ def update_post_counter
     Rails.logger.debug("Ejecutando update_post_counter en Post #{user.id}, user.posts.count: #{user.posts.count}")
     user.update(post_counter: user.posts.count)
   end
```

Comment:
- [ ] [OPTIONAL] Greate job on creating the counter increament method however instead of using the method `update` I would recommend using `increament!`. it is recommended to use the increment! method instead of the update method, as this can improve performance by reducing the number of SQL statements that need to be executed. Example code snippet:

```rb
# app/models/post.rb
...

def update_post_counter
  author.increment!(:posts_counter)
end


```